addons/pos_self_order/static/tests/tours/self_order_mobile_tour.js

```diff
@@ -258,3 +258,30 @@ registry.category("web_tour.tours").add("SelfOrderOrderNumberTour", {
         Utils.checkIsNoBtn("Ok"),
     ],
 });
+
+registry.category("web_tour.tours").add("test_self_order_product_availability", {
+    steps: () => [
+        Utils.checkIsNoBtn("My Order"),
+        Utils.clickBtn("Order Now"),
+        LandingPage.selectLocation("Test-In"),
+        Utils.setProductAvailability("Combo Product 5", false),
+        ProductPage.checkProductOutOfStock("Combo Product 5"),
+        ProductPage.clickProduct("Office Combo"),
+        ProductPage.clickProduct("Combo Product 4"),
+        Utils.clickBtn("Add to cart"),
+        Utils.clickBtn("Checkout"),
+        Utils.setProductAvailability("Office Combo", false),
+        Utils.clickBtn("Pay"),
+        Utils.clickBtn("OK"),
```

Comment:
```suggestion
        add check for product is not available dialog
        Utils.clickBtn("OK"),
```
Also add comments if 3 different steps are performed in single tour